content/documentation/pages/6-recipes/6-multi-platform-deployment/1-multiple-platform-accounts.md

```diff
@@ -0,0 +1,531 @@
+---
+path: 'recipes/multi-platform-deployment/multiple-platform-accounts'
+title: 'Role of Multiple Platform Deployments'
+description: 'A walk-through of multiple platform requirements and the configurations for Cloud Foundry and Kubernetes'
+---
+
+# Role of Multiple Platform Deployments
+
+The goal of this recipe is to unpack the scenarios for when multiple platform deployments becomes a necessity. Let's start
+with the use-cases and then dive into the details of how that can be set up in Spring Cloud Data Flow.
+
+## Use Cases
+
+- For certain use-cases, there is the desire to distinguish the deployment of streaming and the batch data pipelines to
+  a unique environment. For instance, you may want to run a predictive model training routine that requires high memory. That
+  compute is usually defined with specific boundaries, and only the particular workloads are allowed to run on them.
+  In other words, you don't want the regular applications to use the high-compute resource pool and saturate its availability.
+  This is particularly important when you're running machines on a pay-by-use basis, to avoid premium costs.
+
+- Similar to the previous use-case, there might be a need to run the applications closer to where the message broker is
+  (i.e., run the business logic close to where the data is). Doing so can avoid the I/O latency to meet the high throughput
+  and low latency service-level agreements (SLAs). Once again, having to orchestrate a deployment pattern where the streaming
+  applications can be targeted to deploy on the same VMs where Kafka is running could help with the SLAs.
+
+- There's evidence of users using the "single" Spring Cloud Data Flow instance to orchestrate a deployment
+  model in which the streaming and batch data pipelines are deployed and launched to multiple environments. This
+  deployment pattern is primarily curated in order to organize the deployment topologies with well-defined boundaries, where
+  a single SCDF mothership can centrally orchestrate, monitor, and manage the data pipelines.
+
+Above scenarios require Spring Cloud Data Flow to deploy streaming and batch applications with flexible platform
+configurations. Thankfully, though, starting with v2.0, Spring Cloud Data Flow supports the multi-platform deployment
+support. With that, users can declaratively configure the desired number of platforms accounts upfront, and use the
+defined accounts at the deployment time to distinguish the boundaries.
+
+Now that we have the understanding of the use-case requirements, let's review the steps to configure multiple platform
+accounts in Kubernetes and Cloud Foundry.
+
+## Configurations
+
+### Kubernetes
+
+Let's suppose you'd want to deploy a stream with 3 applications to `kafkazone` namespace. Likewise, if you'd want to launch
+a batch-job to `highmemory` namespace, the following configurations can be defined in the SCDF's deployment files.
```

Comment:
"to highmemory" namespace" > "to the highmemory namespace"

Reply:
also, using "highmemory" as the namespace name might be confusing given in the example config below the resources limits for the pod are just being set higher than the other. containers are scheduled onto nodes where they fit based on requests or configuration options for selection/avoidance. when reading "highmemory" in relation to a namespace, it makes it sounds like this namespace "has a lot of memory" so i think its the wrong context

Reply:
OK, to me, it is easy to relate to things, so that I don't have to reason through multiple names to refer to the same thing. Anyway, I will change the namespace to be more descriptive now.